node/src/Pos/Wallet/Web/State/Storage.hs

```diff
@@ -707,8 +708,7 @@ instance Migrate AccountInfo where
       where
         mapAddrs =
             HM.fromList
-          . fmap
-            (\(k, v) -> (unsafeCIdToAddress k, migrate v))
+          . fmap (unsafeCIdToAddress *** migrate)
```

Comment:
Honestly I'd tell hlint to be quiet. I understand the original code but not the new.

Reply:
Actually, seriously, add a general exclusion on that hint to the top level project `HLint.hs`.